apache-rat-core/src/main/java/org/apache/rat/analysis/LicenseCollection.java

```diff
@@ -22,33 +22,37 @@
 import java.util.Collections;
 
 import org.apache.rat.analysis.matchers.AbstractMatcherContainer;
+import org.apache.rat.config.parameters.Description;
 import org.apache.rat.license.ILicense;
 import org.apache.rat.license.ILicenseFamily;
 
 /**
- * A collection of ILicenses that acts as a single License for purposes of Analysis.
+ * A collection of ILicenses that acts as a single License for purposes of
+ * Analysis.
  * <p>
- * This class process each license in turn on each {@code matches(String)} call.  When a match is found the 
- * ILicenseFamily for the matching license is captured and used as the family for this license. If no matching 
- * license has been found the default {@code dummy} license category is used.
+ * This class process each license in turn on each {@code matches(String)} call.
+ * When a match is found the ILicenseFamily for the matching license is captured
+ * and used as the family for this license. If no matching license has been
+ * found the default {@code dummy} license category is used.
  */
+
 class LicenseCollection extends AbstractMatcherContainer implements ILicense {
 
     private static final ILicenseFamily DEFAULT = ILicenseFamily.builder().setLicenseFamilyCategory("Dummy")
```

Comment:
Not sure how this dummy is handled in the report, would NONE be clearer here as the family does actually not do anything?